pkg/quarterdeck/config/config.go

```diff
@@ -50,6 +51,17 @@ type TokenConfig struct {
 	RefreshOverlap  time.Duration     `split_words:"true" default:"-15m"`     // $QUARTERDECK_TOKEN_REFRESH_OVERLAP
 }
 
+// Used by the rate limiter
+// Limit: represents the number of tokens that can be added to the token bucket per second
+// Burst: maximum number of tokens/requests in a "token bucket" and is initially full
+// NOTE: if Burst is not included the config, then ALL REQUESTS WILL BE REJECTED!
+// Ttl: //number of minutes before an IP is removed from the ratelimiter map
+type RatelimitConfig struct {
```

Comment:
@bbengfort - if we don't include a value for `Burst` in the config, it defaults to 0 which means that every single request will get rejected.  Our tests will catch this, but should we set the defaults in `rate_limiter.go` if the values are not set?